jwt/claims_test.go

```diff
@@ -18,12 +18,69 @@
 package jwt
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"gopkg.in/square/go-jose.v2"
 )
 
+type testClaims struct {
+	Subject string `json:"sub"`
+}
+
+func TestCustomClaimsNonPointer(t *testing.T) {
+	priv, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Could probably use the pre-generated keys used for the tests in #99 if you wanted to speed up the tests.